hail/src/main/scala/is/hail/shuffler/SameLengthByteArrayComparator.scala

```diff
@@ -4,7 +4,8 @@ import java.util.Comparator
 
 class SameLengthByteArrayComparator extends Comparator[Array[Byte]] {
   def compare(l: Array[Byte], r: Array[Byte]): Int = {
-    assert(l.length == r.length)
+    if (!(l.length == r.length))
```

Comment:
`!=`?

Reply:
Sure, changed. I've been doing a lot of mechanical messageless assertion to if-throws and figured the `if(!(...))` pattern elucidated what's happening.